engine-precompiles/src/native.rs

```diff
@@ -220,6 +223,56 @@ fn get_nep141_from_erc20(erc20_token: &[u8]) -> AccountId {
     .unwrap()
 }
 
+#[cfg(feature = "contract")]
+fn get_native_erc20_state() -> NativeErc20EngineState {
+    use sdk::io::{StorageIntermediate, IO};
+    let bytes = sdk::near_runtime::Runtime
+        .read_storage(&bytes_to_key(
+            KeyPrefix::Config,
+            NATIVE_ERC20_BRIDGE_STATE_KEY,
+        ))
+        .expect(ERR_NATIVE_ERC20_STATE_NOT_FOUND);
+
+    use borsh::BorshDeserialize;
+    NativeErc20EngineState::try_from_slice(&bytes.to_vec())
+        .expect(ERR_NATIVE_ERC20_STATE_DESERIALIZATION_FAILED)
+}
+
+#[cfg(all(feature = "contract"))]
+fn create_refund_promise(
```

Comment:
@birchmd  Yes, it doesn't relay directly, but it is only used by `fn run` method which is behind the `contract` feature, removing this will produce an "unused" warning, also in this case we need to remove the already used feature here https://github.com/aurora-is-near/aurora-engine/blob/c72399765735f7407397ec21b60e5fb609c44dc4/engine-precompiles/src/native.rs#L12